### ISSUE TITLE
how can i download 320kbps ?

### ISSUE THREAD

**Issue by JesWhite** [AllToMP3/alltomp3-app#38]
hi...
i tried to download working fine...
but i want to download best quality...
everytime downloading 256 kb/s...
how can i download 320kbps?
thanks for your supports.

**Comment by mkanet** [comment]
Yes, I'm also curious about this too.  I always get 256kb/s, not 320kbps.  Thanks for all your work.

**Comment by gregberger** [comment]
@NTag there would be a way using slsk client for node.js. 
I've got a cli prototype working and been strugling to create the UI (I'm thinking of forking your [amazing] project). Thanks for your work

**Comment by MrCricket** [comment]
> So there would be no gain to create 320 kb/s MP3s. Sorry :/ .

What about downloading from Spotify Premium account? (320kbps)
